core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -1,18 +1,182 @@
+          “Please note that learners' overall progress score will decrease if some of the content they've played gets moved to a new exploration.”
+        </div>
+      </div>
+      <div class="modal-buttons">
+        <button class="btn cancel-button"
+                (click)="cancel()">
+                Cancel
+        </button>
+        <button class="btn confirm-reason-button"
+                (click)="confirm()">
+          Confirm and Unpblish
+        </button>
+      </div>
+    </div>
+  </div>
 </div>
-<div class="modal-footer">
-  <button class="btn btn-secondary"
-          (click)="cancel()"
-          type="button">
-          Cancel
-  </button>
-  <button class="btn btn-success e2e-test-close-save-modal-button"
-          (click)="confirm()">
-    Unpublish Story
-  </button>
+
+<div *ngIf="!platformFeatureService.status.SerialChapterLaunchCurriculumAdminView.isEnabled">
+  <div class="modal-header">
+    <p>
+      Are you sure you want to unpublish this story? Unpublishing a story will
+      delete any corresponding translation opportunities from the Contributor
+      Dashboard and reject any related pending translation suggestions.
+    </p>
+  </div>
+  <div class="modal-footer">
+    <button class="btn btn-secondary"
+            (click)="cancel()"
+            type="button">
+            Cancel
+    </button>
+    <button class="btn btn-success e2e-test-close-save-modal-button"
+            (click)="confirm()">
+      Unpublish Story
+    </button>
+  </div>
 </div>
+
+<style>
+  .modal-body {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .reason-select {
+    border: 1px solid black;
+    padding: 5px;
+    width: 35%;
+  }
+
+  .reason-select span {
+    font-weight: bold;
+    text-decoration: underline;
+  }
+
+  .reason-select .unpublishing-dropdown {
+    left: 50%;
+    transform: translate(-50%);
+    width: 90%;
+  }
+
+  .reason-select .selected-reason-text {
+    background-color: rgb(218, 218, 218);
+    margin-top: 5px;
+    padding: 5px;
+    white-space: normal;
+    width: 100%;
+    word-wrap: break-word;
+  }
+
+  .reason-select .unpublishing-dropdown-menu {
+    width: 100%;
+  }
+
+  .reason-select .unpublishing-dropdown-item {
+    cursor: pointer;
+    white-space: normal;
+  }
+
+  .warning-and-confirmation {
+    display: flex;
+    flex-direction: column;
+    width: 60%;
+  }
+
+  .warning-container {
+    border: 1px solid black;
+    padding: 10px;
+  }
+
+  .warning-container span {
+    font-weight: bold;
+    text-decoration: underline;
+  }
+
+  .warning-container .warning-text {
+    font-style: italic;
+    line-height: 100%;
+    margin-top: 5px;
+  }
+
+  .modal-buttons {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 15px;
+  }
+
+  .modal-buttons .cancel-button {
+    background-color: #79c979b8;
+    width: 40%;
+  }
+
+  .modal-buttons .confirm-reason-button {
+    background-color: #f1dd9599;
```

Comment:
Please use a hex color code or rgb(a) color code. Use other properties like opacity to get the required color. Here and elsewhere.

Reply:
Done